### ISSUE TITLE
font name revert to default after image restart and won't use name provided.

### ISSUE THREAD

**Issue by rvillemeur** [pharo-graphics/Bloc#672]
Using this script to display Emoji from 'Noto Color Emoji'. The first time, you get a grid of Emoji (hopefully in color). 
![image](https://github.com/user-attachments/assets/693e4d70-d483-4d91-8687-fe16a783fd4f)

Save and close the image. Reopen it and re-execute the script, you'll get 
![image](https://github.com/user-attachments/assets/f734c753-a34c-4e44-a815-c6fa1330d746)

It look like it will default to Consolas or Cascadia font (the 

![image](https://github.com/user-attachments/assets/5f1ad892-0113-4dd4-999d-b81a0875c150)

Behavior reproduced using Pharo 12, Windows 11 or Linux Fedora 41

```smalltalk
| container |
container := BlElement new
				 border: (BlBorder paint: Color red width: 1);
				 background: (Color white);
				 layout: BlFlowLayout horizontal;
				 constraintsDo: [ :c |
					 c horizontal matchParent.
					 c vertical fitContent ].

"Noto Color Emoji has 2668 different glyph"

16r1F600 to: 16r1F64F do: [ :each |
	| elt |
	elt := BlTextElement text: ((Unicode value: each) asRopedText
					fontSize: 40;
					fontName: 'Noto Color Emoji').
	elt
		padding: (BlInsets all: 5);
		border: (BlBorder paint: Color black width: 1).

	container addChild: elt ].

"container openInNewSpace."
^ container
```

**Comment by rvillemeur** [comment]
after a fresh install of bloc
![image](https://github.com/user-attachments/assets/f443f069-019a-4edc-b7c1-ceaf7cb9d561)

after image restart
![image](https://github.com/user-attachments/assets/5e8c017c-a69a-4d20-a1e0-4ec82ea57334)

Updating the font from system doesn't solve the problem

**Comment by tinchodias** [comment]
Yes, this is not only for Noto Color Emoji. Example with other font:

Load the testing fonts with this script:
```smalltalk
"
AeFontManager resetGlobalInstance.
AeFontManager globalInstance scanDirectory: AeFilesystemResources fontsDirectory
"
```

Then inspect this:
```smalltalk
| text |
text := 'office' asRopedText
			        fontSize: 40;
			        fontName: 'Inria Serif';
			yourself.
BlTextElement text: text.
```

Then Save&Close, and reopen the image. The inspected form will look as with default font (Source Sans Pro), instead of Intia Serif, until you re-scan the tests fontDirectory.

**Comment by tinchodias** [comment]
@rvillemeur my idea to fix this issue is changing the font manager to automatically reload fonts on image startup. The user would have System Settings to be able to:
- disable that automatic reload, 
- manage what are the directories to scan
- which is the default font and default code font (now hardcoded).
I commented details of this idea in Alexandrie's issue 31.

**Comment by rvillemeur** [comment]
Add @Ducasse, If I remember well, font handling was a challenge in previous version of Pharo (and still is in some scenario). He may have some concern to share, given Bloc/Alexandrie is the future graphical stack.

**Comment by rvillemeur** [comment]
I found a workaround in my project, to load the font file directly in the image as a subclass of `AeEmbeddedFontFile`. It's working great, and the font is always available. Looking at the subclass of AeEmbeddedFontFile, by default it only contains SourceSansPro font, the only one remaining after a restart. I agree your idea of automatically reloading font on image startup would be a good one.
